### DISCUSSION TITLE
Installing and configure NHost on a local host

### DISCUSSION THREAD

**Discussion by cesarverdesslots** [nhost/nhost#34] [Q&A] [answered]
Sorry for my silly question. Is an step by step detailed explanation for installing and configure NHost and all its dependencies on a local Ubuntu machine?
Thanks a lot!

**Comment by jmarbutt** [comment, marked as answer]
The main dependencies are:

- Docker
- [Hasura CLI](https://hasura.io/docs/latest/graphql/core/hasura-cli/install-hasura-cli.html)

Those are the main two, not sure if there are others that I am missing off the top of my head.

**Reply by cesarverdesslots** [reply]
Thank you very much. I got it up and running on my ubuntu with your help.

**Reply by chymian** [reply]
hey @cesarverdesslots,
can you pls. provide a little how-to for the setup you have done?
TIA
